corehq/ex-submodules/couchforms/tests/test_edits.py

```diff
@@ -274,7 +274,7 @@ def test_case_management_ordering(self):
 
         if not settings.TESTS_SHOULD_USE_SQL_BACKEND:
             self.assertEqual(
-                [create_form_id, edit_form_id, second_edit_form_id],
+                [create_form_id, create_form_id, edit_form_id, second_edit_form_id],
```

Comment:
As found out with Cory, this is needed since date_opened is added now which results in an additional update block on the create_form causing create_form_id to repeat here.
@czue : is that quoted correctly?